filetests/verifier/isplit-bb.clif

```diff
@@ -0,0 +1,28 @@
+test compile
```

Comment:
Can you add test annotations to check that the compiled form is what we expect, here, please?

Reply:
Actually we can't check for the split instruction being present in the output, because we can reuse a pair created by concat, it seems.